### PR TITLE
Css grid refactor

### PR DESCRIPTION
**Pull request by sebasdeldi**
Added mobile and tablet views for detail view using css grids
Tablet view:
image
Mobile view:
image

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -37,97 +36,152 @@ $detail-margin-bottom: 30px;
...
-            }
-
-            .label {
-              @include bold-text($attribute-font-size * 0.7);
-            }
-          }
-        }
-      }
-    }
+		}
```
Comment (jjuli93) [inline scss/detail.scss:39]: check indentation on this file

Comment (sebasdeldi) [reply scss/detail.scss:39]: hmm it kinda looks like some sort of github visual bug, the file seems fine in my editor.
image

File: scss/detail.scss [hunk 2]
```diff
@@ -37,97 +36,152 @@ $detail-margin-bottom: 30px;
...
+		grid-template-rows: auto auto;
+		grid-template-areas:
+			'header header'
+			'image-container detail'
+		;
+		padding: 32px;
+
+		@include sm {
+			grid-template-columns: 1fr;
+			grid-template-rows: auto auto auto auto;
```
Comment (jjuli93) [inline scss/detail.scss:74]: i think theres no need to define this, with grid-template-areas is enough

File: scss/detail.scss [hunk 3]
```diff
@@ -37,97 +36,152 @@ $detail-margin-bottom: 30px;
...
+			;
+			display: grid;
+		}
+
+		.header {
+			grid-area: header;
+			text-align: center;
+			display: grid;
+			grid-template-columns: 1fr;
+			grid-template-rows: auto auto;
```
Comment (jjuli93) [inline scss/detail.scss:89]: same here

File: scss/detail.scss [hunk 4]
```diff
@@ -37,97 +36,152 @@ $detail-margin-bottom: 30px;
...
+			}
 
       .cover-image {
         width: 100%;
-      }
+			}
+			
+			@include sm {
+				grid-template-columns: 1fr;
+				grid-template-rows: auto auto auto;
```
Comment (jjuli93) [inline scss/detail.scss:176]: same here

**Pull request merged**